core/imds/src/main/java/software/amazon/awssdk/imds/Ec2MetadataRetryPolicy.java

```diff
@@ -122,16 +122,14 @@ public interface Builder extends CopyableBuilder<Ec2MetadataRetryPolicy.Builder,
          */
         Builder numRetries(Integer numRetries);
 
-        @Override
-        Ec2MetadataRetryPolicy build();
     }
 
-    private static final class BuilderImpl implements Builder {
+    public static final class BuilderImpl implements Builder {
```

Comment:
Hmm, why do we need it to be public?

Reply:
we need to instantiate a Ec2MetadataRetryPolicy.Builder for the `retryPolicy(Consumer<Ec2MetadataRetryPolicy.Builder> builderConsumer)`

Reply:
It seems we can just use `applyMutation` instead of instantiating the builder class.
```
Ec2MetadataRetryPolicy.builder().applyMutation(builderConsumer)? 
```
https://github.com/aws/aws-sdk-java-v2/blob/master/services-custom/s3-transfer-manager/src/main/java/software/amazon/awssdk/transfer/s3/S3TransferManager.java#L209

Reply:
oh nice, yeah that works 👍

Reply:
Oh wait, we dont even need applyMutation, I can just use `Ec2MetadataRetryPolicy.builder()` 🤦